components/tabs/tab-mixin.js

```diff
@@ -92,25 +92,22 @@ export const TabMixin = superclass => class extends SkeletonMixin(superclass) {
 
 	render() {
 		return html`
-			<div class="d2l-skeletize">
```

Comment:
Removing this as I'll add the skeleton piece later and this didn't actually work as expected.